### DISCUSSION TITLE
Whether the anomaly image will be involved in the validation process

### DISCUSSION THREAD

**Discussion by 1094558673** [open-edge-platform/anomalib#2699] [General]
Hello, I would like to ask whether the anomaly image will be involved in the validation process during training if the default parameter in config is followed, that is, whether there is anomaly data in the validation set.

test_split_mode: from_dir # options: [from_dir, synthetic]
test_split_ratio: 0.2 # fraction of train images held out testing (usage depends on test_split_mode)
val_split_mode: same_as_test # options: [same_as_test, from_test, synthetic]
val_split_ratio: 0.5 # fraction of train/test images held out for validation (usage depends on val_split_mode)

**Comment by rajeshgangireddy** [comment]
Hi @1094558673 
Yes, anomalous images are used for validation (but never for training).
If you would like to know more about how train/test/val images are used in anomalib see this comment :  
 https://github.com/open-edge-platform/anomalib/issues/1067#issuecomment-2024908589
